src/test/java/uk/gov/hmcts/reform/bailcaseapi/domain/entities/ccd/StateTest.java

```diff
@@ -14,14 +14,13 @@ void has_correct_values() {
         assertEquals("applicationEnded", State.APPLICATION_ENDED.toString());
         assertEquals("applicationSubmitted", State.APPLICATION_SUBMITTED.toString());
         assertEquals("bailSummaryUploaded", State.BAIL_SUMMARY_UPLOADED.toString());
-        assertEquals("uploadBailSummary", Event.UPLOAD_BAIL_SUMMARY.toString());
-        assertEquals("recordTheDecision", Event.RECORD_THE_DECISION.toString());
+        assertEquals("signedDecisionNoticeUploaded", State.SIGNED_DECISION_NOTICE_UPLOADED.toString());
         assertEquals("unknown", State.UNKNOWN.toString());
     }
 
     @Test
     void fail_if_changes_needed_after_modifying_class() {
-        assertEquals(8, State.values().length);
```

Comment:
The number should match number of assertEquals tested in 'has_correct_values'. Think if you add 'recordTheDecision' back in, 9 is fine.